presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcResultSetTimezone.java

```diff
@@ -172,6 +175,75 @@ public void testTimestamp(Optional<String> sessionTimezoneId)
         });
     }
 
+    @Test(dataProvider = "timeZoneIds")
+    public void testTimestampWithTimeZone(Optional<String> sessionTimezoneId)
+            throws Exception
+    {
+        checkRepresentation(
+                "TIMESTAMP '1970-01-01 00:00:00.000 +00:00'", // Presto
+                ImmutableList.of(
+                        "TIMESTAMP WITH TIME ZONE '1970-01-01 00:00:00.000 +00:00'", // PostgreSQL
+                        "from_tz(TIMESTAMP '1970-01-01 00:00:00.000', '+00:00')"), // Oracle
+                TIMESTAMP,
```

Comment:
Why not `TIMESTAMP_WITH_TIMEZONE`? Todo?

Reply:
fixed